examples/task-example.yaml

```diff
@@ -49,6 +49,8 @@
     type: RANDOM-SEARCH # will be normalized
     execution-batch-size: 42
     execution-count: 420
+    maximum-queued-executions: 17
+    on-child-error: stop_all-AND_error
```

Comment:
Opinion: Is this written intentionally wrong to represent fail-safe working in `cast` method? Ok, but maybe bit weird for an example file 🤔 I would consider adding weirdness into separate test file and merging those into better written example

Reply:
Maybe I'll add a comment there too, sure :)